ops/terraform/modules/resources/bfd_cw_dashboards/templates/bfd-dashboards.tpl

```diff
@@ -204,8 +204,8 @@
                 ],
                 "view": "singleValue",
                 "region": "us-east-1",
-                "title": "HTTP 500s",
-                "period": 300,
+                "title": "Total number of HTTP 500s errors in the last 24 hours",
```

Comment:
it is better, but the title will be wrong once the period is changed by the dashboard user, right? are there any solutions to that? can the title be parametrized by the period?

Reply:
The title won't be wrong if the dashboard user change the range of the entire dashboard it will still show last 24 hours within "some" range that the user will select. However the title will be wrong if the user edited the widget and alter the period.